src/coreComponents/dataRepository/DataContext.hpp

```diff
@@ -62,12 +61,11 @@ class DataContext
   string getTargetName() const
   { return m_targetName; }
 
-  /**
-   * @brief Flag on availability of file information. Used to provide more user-friendly information.
-   * @return true if the context is from a file.
-   */
-  bool isDataFileContext() const
-  { return m_isDataFileContext; }
+  virtual string getTargetNameInPath( bool & fileLineFound ) const
```

Comment:
Can you explain the `fileLineFound`? I do not really understand why we need it.

Reply:
The `fileLineFound` helps to know if the target name contains file line infos. It is used by `GroupContext::toString()`.
More info in this reply: https://github.com/GEOS-DEV/GEOS/pull/2404/#discussion_r1250949982